pyccl/core.py

```diff
@@ -562,6 +571,9 @@ def _build_parameters(
                     "of neutrino masses, only with a sum." % m_nu_type)
             elif m_nu_type is None:
                 m_nu_type = 'list'  # False
+            mnu_list = [0]*3
+            for i in range(0,3):
+                mnu_list[i] = m_nu[i]
```

Comment:
What is the purpose of this block of code?

Reply:
If m_nu is passed as a sum, m_nu is a single value and it gets split into 3 values which are held in mnu_list below. If m_nu is passed as a list, this bit of code just puts those individual listed values in mnu_list so that below we can work with mnu_list regardless of the way m_nu is initially provided. I agree it's clunky though - better ideas?

Reply:
Ahhhhh. I did not follow. This is totally fine. No need for a change here. Thanks!